extension/src/popup.js

```diff
@@ -94,66 +110,99 @@ stopToonin.onclick = () => {
     });
 }
 
+createRoomBtn.onclick= () => {
+    createRoomBtn.style.display = "none";
+    connectRoomBtn.style.display = "none";
+    shareButton.style.display = "block";
+    roomNameSpan.style.display = "block";
+    btnBack.style.display = "block";
+
+}
+
+connectRoomBtn.onclick= () => {
+    createRoomBtn.style.display = "none";
+    connectRoomBtn.style.display = "none";
+    roomNameToonin.style.display = "block";
+    btnBack.style.display = "block";
+    playButton.style.display = "block";
+    connectSpan.style.display = "inline-flex";
+
+}
+
 chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
     var activeTab = tabs[0];
     chrome.tabs.sendMessage(activeTab.id, {"message": "clicked_extension"});
 });
 
   chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
     if (request.message === "extension_state_from_background" && request.data.roomID) {
-        roomNameSpan.style.visibility= "hidden";
-        shareButton.style.visibility= "hidden";
-        stopSharingButton.style.visibility="visible";
-        copyButton.style.visibility="visible";
```

Comment:
Maybe you can send a call to the create room button function (line 113)

Reply:
Hey thanks for the feedback adresssing them right now.
For 3, the stop button is visible when you connect to someone elses room.